src/components/SettingsCheckbox/SettingsCheckbox.vue

```diff
@@ -0,0 +1,81 @@
+<!--
+  - @copyright Copyright (c) 2019 Julius Härtl <jus@bitgrid.net>
+  -
+  - @author Julius Härtl <jus@bitgrid.net>
```

Comment:
@GretaD Add your own copyright, not Julius' :P

Reply:
I added myself only on the documents that i edited myself. this one I got it from richdocuments done by Julius and didnt really change anything, so I thought would be fair this way :)